<launch>

    <include file="$(find mbot_simulation)/launch/mbot_gazebo.launch"/>

    <include file="$(find mbot_simulation)/launch/mbot_sim_control.launch"/>

    <group ns="xtion">
        <node name="depth_to_laser" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan">
            <remap from="image" to="/camera/depth/image_raw"/>
            <remap from="camera_info" to="/camera/depth/camera_info"/>
            <param name="scan_height" value="20"/>
            <param name="output_frame_id" value="/xtion_link"/>
        </node>
    </group>

</launch>